File: py2many/async_await.go
package main

import (
"fmt")



#[async]
func task() {
fmt.Printf("%v\n","task started");
await!(asyncio.sleep(5));
fmt.Printf("%v\n","task finished");}


func Main() {
task1 := asyncio.create_task(task())
fmt.Printf("%v\n","task created");
await!(task1);
fmt.Printf("%v\n","done");}

File: py2many/async_await.jl
#[async]
function task()
println(join(["task started"], " "));
await!(sleep(asyncio, 5));
println(join(["task finished"], " "));
end

function main()
task1 = create_task(asyncio, task())
println(join(["task created"], " "));
await!(task1);
println(join(["done"], " "));
end

